src/quo2/components/profile/collectible/view.cljs

```diff
@@ -0,0 +1,99 @@
+(ns quo2.components.profile.collectible.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.profile.collectible.style :as style]
+            [react-native.core :as rn]))
+
+(defn remaining-tiles
+  [amount]
+  [rn/view {:style (merge style/bottom-right (style/remaining-tiles))}
+   [text/text
+    {:style  (style/remaining-tiles-text)
+     :size   :paragraph-2
+     :weight :medium}
+    (str "+" amount)]])
+
+(defn tile
+  [{:keys [style resource size]}]
+  (let [source (cond->> resource
```

Comment:
this pretty hard to read
```clojure 
(if (string? resource) {:uri resource} resource)
```
regural if is more readable imo

Reply:
Agree, I changed it.

Also, just wanted that comment that IMO the `cond->` is more readable when it's aligned in pairs:
```clojure
(cond-> v
  a fn-1
  b fn-2
  c fn-3)
```
But the lint-fix is aligning it as follows:
```clojure
(cond-> v
        a
        fn-1
        b
        fn-2
        c
        fn-3)
```

which is a little confusing.